data_management/opensearch_indexer/opensearch_indexer/text_extraction.py

```diff
@@ -7,11 +7,40 @@
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
+SUPPORTED_TEXTRACT_FORMATS = [
```

Comment:
so is this all the formats it supports?

Reply:
also which ones do we not add tests for?

Reply:
@anthonyhashemi yes should be all of them, taken from this list

<img width="761" alt="image" src="https://github.com/user-attachments/assets/be2d29a7-3cb0-45fb-a45f-61eb5ddae9bc">

At the moment tests dont cover the ones that require system dependencies like Tesseract or antiword

Reply:
Can you create a ticket for writing e2e tests for those please. Not to do now. We need to think of the best way to run e2e tests for the indexing.

Reply:
@anthonyhashemi sure, created a ticket